### PR TITLE
init-project

### PR DESCRIPTION
**Pull request by hungvv-1301**
### What ? ###
- Feature #56445 Init project
- Task #56446 Apply MVVM Structure

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (minhnn-1279) [inline app/.gitignore:1]: check endline nhé :v

File: app/src/main/java/com/atom/android/lebo/base/BaseAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                object : RecyclerView.OnScrollListener() {
+                    override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
+                        super.onScrolled(recyclerView, dx, dy)
+                        val linearLayoutManager = recyclerView.layoutManager as LinearLayoutManager?
+                        val sizeData = recyclerView.adapter?.itemCount?.minus(1)
+                        if (
+                            linearLayoutManager != null &&
+                            linearLayoutManager.findLastCompletelyVisibleItemPosition() ==
+                            sizeData &&
+                            sizeData != -1
```
Comment (minhnn-1279) [inline app/src/main/java/com/atom/android/lebo/base/BaseAdapter.kt:29]: extract constant

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (minhnn-1279) [inline app/src/main/res/layout/activity_main.xml]: extract string

**Review by hungld-1302 — APPROVED** [review]

**Review by ndt0409 — APPROVED** [review]

**Review by nhanviet1 — APPROVED** [review]

**Review by daolq-2712 — APPROVED** [review]
OK!

**Pull request merged**